src/routes/blog/post/announcing-init/+page.markdoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Announcing Init, what's next?"
+description: 
+date: 2024-02-19
+cover: /images/blog/init-announcement.png
+timeToRead: 4
+author: laura-du-ry
+category: open-source
+---
+
+We are very excited to finally announce Init. 
+
+A new way to experience Appwrite releases, starting with Appwrite 1.5.
```

Comment:
Not just releases. The focus is around release, updates, news, content. Everything that is new in Appwrite. Init is the start of something new.

Reply:
Updated to: A new way to experience everything new with Appwrite, starting with Appwrite 1.5.